### ISSUE TITLE
Update normalization notebook to work with automatic voltage updates

### ISSUE THREAD

**Issue by ngreenwald** [angelolab/toffy#367]
**Is your feature request related to a problem? Please describe.**
The current normalization notebook calculates the MPH per image over the course of a run, fits a curve to that decay, and uses that to normalize each channel in each image. Ionpath is going to be rolling out an update to the instrument which automatically adjusts the detector voltage over the course of the run. This will significantly reduce the amount of signal decay we see over the course of the run. 

**Describe the solution you'd like**
A side effect of this will be that our current approach for normalizing the data will need to be updated. Specifically, we'll probably want to make a separate notebook, updated_normalization or something like that, since the workflow will be different. Rather than fitting a polynomial, we'll want to check that all of the values are roughly on a straight line. We should then use that fitted flat line as the normalization value for all the FOVs for that particular channel. 

The outlier detection should instead look for values that are far away from that flat line. Some of the other checks (like making sure the voltage in the JSON doesn't change) will need to be tweaked as well.

**Comment by alex-l-kong** [comment]
We can use `sklearn.LinearRegression` (or another standard linear regression library) to generate the fit. How many SDs away from the line of best fit do we want to use for determining outliers?

**Comment by ngreenwald** [comment]
We'll need to take a look at some real data and then decide
